lib/studies/reply/adaptive_nav.dart

```diff
@@ -816,7 +826,7 @@ class _ReplyFab extends StatelessWidget {
                 style: Theme.of(context)
                     .textTheme
                     .headline5
-                    .copyWith(fontSize: 16),
+                    .copyWith(fontSize: 16, color: ReplyColors.black900),
```

Comment:
Is this always supposed to be black regardless of light or dark mode?

Reply:
Yes black text on the fab is always visible since the fab is colored.